### DISCUSSION TITLE
does anyone have the same issue?

### DISCUSSION THREAD

**Discussion by mattiadanisiDev** [community/community#111135] [GitHub Education]
### Hi everyone,

I cant sign for github student. When I have to upload a proof that I am currently a student and then submit the photo of my student ID, It reloads and redirect me on the same page where I have to upload the student ID. It keeps like this forever. Can anyone relate?

**Comment by SebastienXia** [comment]
### Hello, @mattiadanisiDev 

>I had the same problem, the file download does not work, you just have to take a photo with your phone to be able to finish the process.

[close the discussion and mark as answered if this helped you with your problem]

**Comment by samus-aran** [comment]
Hi @mattiadanisiDev 

Are you able to provide more details:

- Console log errors (via your Browser developer tools)
- Screen-recording or screenshots

This will help us troubleshoot the issue you're experiencing. 👍

**Comment by mishmanners** [comment]
The same is resolved here: https://github.com/orgs/community/discussions/88401#discussioncomment-8142514 and here: https://github.com/orgs/community/discussions/88446#discussioncomment-8148738

Please see these. The TL;DR:

- Refresh the page or clear your browser's cache and cookies
- Check for browser extensions that could be interfering with the JavaScript code and disable them
- Try a different browser
- Ensure your file is less than 150kB
